doc/guide/feature-realmd.xml

```diff
@@ -24,4 +24,12 @@ $ <command>realm join example.com</command>
 Password for Administrator:
 </programlisting>
 
+<para>
+<ulink url="http://www.freedesktop.org/software/realmd/">realmd</ulink>
+sets up domain-qualified user names by default, i. e. login user names look like
+"<code>user@example.com</code>". For using unqualified names (just
+"<code>user</code>"), set the <code>fully-qualified-names</code> option in
+<ulink url="https://www.freedesktop.org/software/realmd/docs/realmd-conf.html">/etc/realmd.conf</ulink>.
```

Comment:
Does this need to be set before joining?  Would it actually make sense to make this an option in our dialog?

(I can't say what the advantages/disadvantages are of either...)

Reply:
In that file it needs to be set before joining, yes. After joining one can edit a different, but similar file in /etc/ipa/* (or so); I tried that and it works, but this is what the IPA authors suggested. I already mentioned the possibility of adding a checkbox to the dialog in the commit message, depending on the outcome of the upstream discussion.

Reply:
*ack*

Reply:
Updated the documentation to say "before joining".